src/components/SessionOverview.vue

```diff
@@ -83,14 +83,14 @@ export default {
     setSessions(sessions) {
       if (Array.isArray(sessions) && sessions.length > 0) {
         let isOpen = (session) => {
-          if (!session.wrapSignature) return true;
-          //if (!(session.claimed ?? false)) return true;
+          if (!session.wrapSignatureA) return true;
+          if (!session.wrapSignatureB) return true;
           return false;
         };
 
         this.mysessions = sessions.map((session) => {
           return {
-            status: isOpen(session) ? "open" : "closed",
+            status: isOpen(session) ? "open" : "open",
```

Comment:
This is not a mistake. Sessions were closed when I expected them to be open, so I've just set them to always being open. Let's discuss this and agree on what the definition of an open session is now.